examples/consume.ts

```diff
@@ -0,0 +1,95 @@
+import Fluvio, {
+    FetchLogOption,
+    FetchOffset,
+    Isolation,
+    ReplicaKey,
```

Comment:
I don't think that we need to expose the term `Replica` in our public APIs. We don't have any documentation that introduces that term with a solid foundation, and even if we did I'm not convinced it's something a user needs to know about. A ReplicaKey is nothing more than just a topic and partition number. The Rust API just replaced all public usages of a ReplicaKey with just two arguments for topic and partition and that made everything look nice and clean.